src/functionalTest/java/uk/gov/hmcts/reform/prl/controllers/managedocuments/ManageDocumentsControllerFunctionalTest.java

```diff
@@ -64,6 +66,19 @@ public void givenManageDocuments_whenCopy_manage_docsEndPoint_thenRespWithCopied
             .assertThat().statusCode(200);
     }
 
+    @Test
+    public void givenManageDocuments_GiveErrorWhenCourtAdminUserSelectCourt() throws Exception {
+        String requestBody = ResourceLoader.loadJson(MANAGE_DOCUMENT_COURT_REQUEST);
+        request
+            .header("Authorization", idamTokenGenerator.generateIdamTokenForSolicitor())
+            .body(requestBody)
+            .when()
+            .contentType("application/json")
+            .post("/manage-documents/validate-court-user")
+            .then()
+            .assertThat().statusCode(200);
```

Comment:
can response data also checked rather than just status code?

Reply:
@shashisk97 added check for response verification